slf4j-bridge/src/test/scala/zio/logging/slf4j/bridge/Slf4jBridgeSpec.scala

```diff
@@ -83,6 +83,11 @@ object Slf4jBridgeSpec extends ZIOSpecDefault {
             )
           )
         )
-      }.provide(Slf4jBridge.initialize)
-    )
+      },
+      test("Implements Logger#getName") {
+        for {
+          logger <- ZIO.attempt(org.slf4j.LoggerFactory.getLogger("zio.test.logger"))
+        } yield assertTrue(logger.getName == "zio.test.logger")
+      }
+    ).provide(Slf4jBridge.initialize) @@ TestAspect.sequential
```

Comment:
The first case fails when I change `proved` to `provideShared` or remove `TestAspect.sequential`. Do you have any idea of the reason?

Reply:
It seems like expected behavior of `ZTestLogger.default`.